app/actions/FileActions.ts

```diff
@@ -1,6 +1,7 @@
 import slug from 'slugify';
+import { imageGallerySchema } from 'app/reducers';
 import type { Thunk } from 'app/types';
-import { File as FileType } from './ActionTypes';
+import { File, File as FileType, ImageGallery } from './ActionTypes';
```

Comment:
```suggestion
import { File as FileType, ImageGallery } from './ActionTypes';
```
I think the type `UploadArgs.file` which is set to `File` on line 42 is intended to be the built-in [DOM-File type](https://developer.mozilla.org/en-US/docs/Web/API/File). Importing `File` here overshadows this, which is probably why it was imported as `FileType` earlier. 

If you just remove `File` from the import and change the action-type: `types: File.PATCH` to `types: FileType.PATCH` on line 109, I believe all the types will be correct.